test/services/ProveCoGateway/proveCoGateway.test.ts

```diff
@@ -0,0 +1,241 @@
+import BigNumber from 'bignumber.js';
```

Comment:
These test cases will change if proveCoGateway method is made private.

Reply:
Tests are updated